fluentui_transients/src/main/java/com/microsoft/fluentui/tooltip/Tooltip.kt

```diff
@@ -295,12 +302,12 @@ class Tooltip {
         val startPosition = positionX + layoutParams.marginStart
         val topBarHeight = context.statusBarHeight + ( context.activity?.supportActionBar?.height ?: 0 )
         val doesNotFitAboveOrBelow = (positionY < topBarHeight) || (positionY + contentHeight > displayHeight)
-        val rightSpace = displayWidth - anchorRect.right - context.softNavBarOffsetX
-        val rightEdge = ( startPosition + upArrowWidth + cornerRadius + margin > displayWidth ) || (doesNotFitAboveOrBelow && anchorRect.left > rightSpace)
+        val rightSpace = displayWidth - anchorRect.right + context.softNavBarOffsetX
```

Comment:
Why are we changing this?
If we are calculating it from right and the navbar is shown in right then it should be subtracted?

Reply:
We calculate the softNavBarOffset only at 270degrees when navbar is on the left. 
These were producing errors in my application, have tested this change there.